CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -39,12 +39,12 @@ internal static class JsonFileDalUtils
         /// <summary>
         /// A remark to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
+        public const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
```

Comment:
I would suggest naming both variables using the style `DefaultExchangeHeaderRemark` and `DefaultExchangeHeaderCopyright`.